src/odemis/acq/stream/_sync.py

```diff
@@ -80,7 +80,7 @@ class MultipleDetectorStream(with_metaclass(ABCMeta, Stream)):
     the .raw from the sub-streams.
     """
 
-    def __init__(self, name, streams):
+    def __init__(self, name, streams, md_manager=None):
```

Comment:
If you use the stream.acquire() function to store the metadata, you don't need to pass the md_manager to every stream (which would be a good thing), and instead just pass it to the `acquire()` function.

Reply:
indeed, that's much better!